debian/changelog

```diff
@@ -11,8 +11,9 @@ kano-desktop (3.14.0-0) unstable; urgency=low
   * Added fast bootup animation using kano-splash
   * Matched Chromium icon on the Desktop with the one on Dashboard
   * Change to use kano-start-browser
+  * Fix for low battery signal on boot software regression
```

Comment:
I don't know if you need that here, wasn't the regression introduced since the last release?

Reply:
That's true. I'll remove that.